### PR TITLE
NASA-BalanceBranch

### PR DESCRIPTION
**Pull request by vampirika**
### Quel'thalas:

- Increase spellbreaker control magic cost 0.2 -> 0.5 mana per health.

### Gilneas:

- Removed resistant skin from worgen.

### Scourge:

- Increase anub'arak beetle health.
  - Level 1 450 -> 750
  - Level 2 650 -> 950
  - Level 3 850 -> 1150
- Increase anub'arak carapce armour, added damage reduction.
  - Armour increased from 5/7/10/14 -> 5/10/15/20.
  - NEW Anub'arak damage taken now 100/90/80/70%.
- Increased scale size of crypt burrow. Reduced crypt fiend burrow hp to match crypt fiend.
- Scourge early creeping was too strong, so the starting ghoul has been removed.

### Legion:

- Increase legion archer primary attack.
  - Damage increased 18-21 -> 28-31.
  - Burning arrows ability damage increased 5 -> 10; duration decreased 5 -> 4. (_total damage increased 25 -> 40)
- Increased spellcaster damage output.
  - Succubus mindflay damage per second increased 10 -> 25.
  - Eredar flame shield damage per second increased 15 -> 20.
  - Eredar attack damage increased 11-14 -> 16-20.

### Stormwind:

- Increased chaplain attack damage 8-10 -> 13-15.

### Kul'tiras:

- Bombers more of an investment to build and engage in fights.
  - Bomber and cluster rocket upgrade both now require flight.
  - Cluster rocket cast range decreased 800 -> 600.

### Fel Horde:

- Units from blackrock will no longer attack redridge on quest completion. A ping occurs on quest completion to notify player of unlock.
- Obsidian drake stronger
  - Gold cost decreased 32 -> 30.  
  - Health increased 1025 -> 1175.
  - Attack damage increased 36-47 -> 41->52.
  - Attacks now cause damage in a small AoE - Full damage (50)/Medium damage (75)/Small damage (100).

### CODE REVIEW

**Comment by YakaryBovine** [comment]
The new Spiked Carapace behaviour needs to be properly reflected in its tooltip.

The Learn tooltip needs to indicate that it gives damage reduction in the core description, not just in the per-level ones. So something like "Increases the hero's armor, and causes it to take reduced damage from all sources and return damage to enemy melee attackers." Also, just take out the level 1 bit where it says it takes 100% damage - that's not a function of the ability.
image

The wording on the normal tooltip is pretty rough. Right now it just says that it's "taking 90% damage". Which, common sense notwithstanding, could mean anything. Refer to the learn tooltip I provided above for an example of being a little more verbose.
image

File: mapdata/WarcraftLegacies/UpgradeData/Skin/909324370.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Improves Obsidian Drakes by causing their attacks to deal \u003CA0DS,DataA1\u003E additional damage on the first attack, twice as much on the second attack, three times as much on the third attack, etc.|n|nIf a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
+      "Value": "Improves Obsidian Drakes by causing their attacks to deal cause fire to cling to a unit. If a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UpgradeData/Skin/909324370.json:46]:
```suggestion
      "Value": "Improves Obsidian Drakes by causing their attacks to cause fire to cling to a unit. If a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
```
Also FYI these Incinerate tooltip changes will conflict with the similar fix I did in dev, but that's OK - I'll just keep dev for those.

File: mapdata/WarcraftLegacies/AbilityData/Skin/1396781121.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Forms a shield of flames around a target unit, dealing 15 damage per second to units around it. |nLasts 20 seconds. Lasts 10 seconds on heroes."
+      "Value": "Forms a shield of flames around a target unit, dealing \u003CA0AS,DataA1\u003E damage per second to units around it. |nLasts \u003CA0AS,Dur1\u003E seconds. Lasts \u003CA0AS,HeroDur1\u003E seconds on heroes."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1396781121.json:38]: Good shit!!!

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Awesome changes, just need more thought on the Spiked Carapace tooltip.

**Comment by vampirika** [comment]
Updated carapace tooltip for extra clarity. There is now a conflict but I think it's one that doesnt matter?

**Review by YakaryBovine — APPROVED** [review]
Cool, looking good!

**Review by YakaryBovine — APPROVED** [review]
LGTM

**Pull request merged**